modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp

```diff
@@ -320,10 +320,23 @@
 <clay:row>
 	<clay:col
 		id="image-card-block"
-		md="6"
+		md="4"
+	>
+		<clay:user-card
+			name="User Name"
+			subtitle="Latest Action"
+			userColorClass="info"
+		/>
+	</clay:col>
+
+	<clay:col
+		id="image-card-block"
+		md="4"
 	>
-		<clay:user-card-v2
+		<clay:user-card
 			actionDropdownItems="<%= cardsDisplayContext.getActionDropdownItems() %>"
+			icon="picture"
+			labels="<%= cardsDisplayContext.getLabelItems() %>"
```

Comment:
Isn't this `getLabels`?

Reply:
Nope, in [CardsDisplayContext](https://github.com/liferay/liferay-portal/blob/e069efe223205daa68feca93d7908445b5c8c69a/modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/CardsDisplayContext.java#L53) the defined that method as `getLabelItems`

Reply:
Ok... maybe I got confused by https://github.com/liferay-frontend/liferay-portal/pull/434/files#diff-73750b5c26581ba24bc65db95e69e429b13711fbed5cee5f1fd9ad6f221a03a6R64 since I'm only seeing this in GitHub UI... 🤔

Reply:
Yeah, that's the new `UserCardsDisplayContext` I created, which implements `UserCard`. The one you're pointing is the old `cardsDisplayContext` which implements nothing and have `getLabelItems` method. I can try to update all `user-card` usages to use the same model instead this old `cardsDisplayContext`

Reply:
So, in the `taglib-clay-sample-web` we now have 2 different `CardDisplayContext` classes, one free, one implementing `UserCard`?

I think the convention for the latter is to name them `UserCard` and not `DisplayContext`, and then assign it as `userCard="<%= userCard %>` rather than by exploding the attributes. Maybe that's what's confusing me... 🤔

Reply:
Ok, I'll rename it to `UserCard`

Reply:
> So, in the taglib-clay-sample-web we now have 2 different CardDisplayContext classes, one free, one implementing UserCard?

Yes. Once we migrate all card tags we should be able to get rid of that `CardDisplayContext` because we'll be using our own models for each tag (`UserCard`, `HorizontalCard`, `VerticalCard`, etc)